Library/Formula/sfk.rb

```diff
@@ -0,0 +1,22 @@
+class Sfk < Formula
+  homepage "http://stahlworks.com/dev/swiss-file-knife.html"
+  url "http://internode.dl.sourceforge.net/project/swissfileknife/1-swissfileknife/1.7.4/sfk-1.7.4.tar.gz"
+  version "1.7.4"
```

Comment:
Version is redundant here. The URL is simple enough that ` brew ` can automatically read the version from it.